src/main/java/org/folio/marccat/resources/HeaderTypeAPI.java

```diff
@@ -29,4 +34,21 @@ public HeadingTypeCollection getHeadingTypes(
       return null;
     }, tenant, okapiUrl, configurator);
   }
+
+  @GetMapping("/auth-header-types")
+  public HeadingTypeCollection getAuthorityHeadingTypes(@RequestParam final String code,
+      @RequestParam final String lang, @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) final String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl) {
+    return doGet((storageService, configuration) -> {
+
+      HeadingTypeCollection headingTypeCollection = new HeadingTypeCollection();
+      List<HeadingType> headingTypes = new ArrayList<>();
+      HeadingType headingType = new HeadingType();
+      headingType.setCode(Global.AUT_MATERIAL_DESCRIPTION_HEADER_TYPE);
+      headingType.setDescription(Global.AUT_MATERIAL_DESCRIPTION_DESC);
+      headingTypes.add(headingType);
+      headingTypeCollection.setHeadingTypes(headingTypes);
+      return headingTypeCollection;
```

Comment:
What is going on here? It looks like we are returning a canned response and not using the storage service to look this up. If that is the case, what is the point of this API? It does not seem that these values are configurable since they are constants. Why can't the UI do this? It seems like a waste of resources to retrieve something that will never change. Maybe I am misunderstanding this code...

Reply:
The UI cannot solve the code so it needs this API

Reply:
I'm not sure what you mean. The only thing we are setting here are constants. Why can't the UI know these constants? 

Why call `doGet` if neither `storageService` or `configuration` are used to build the `HeadingTypeCollection`?

Reply:
We use it, because in the future we will use this function to get more values from database